client/src/components/CustomTables.js

```diff
@@ -71,7 +75,7 @@ function formatDate(unformattedDate) {
 export function PastPracticeTable(props) {
   const classes = pastPracticeTableStyles();
   const [completedInterviewsList, setCompletedInterviewsList] = useState([]);
-
+  const [inLobby, setInlobby] = useState(true);
```

Comment:
Let's be consistent with camelCase throughout the app - so setInLobby instead of setInlobby